src/dcm230/dcm230.py

```diff
@@ -190,8 +216,8 @@ def _read_register(self, register_name: str) -> Decimal | int:
             return round(value, spec.decimals)
         return self._unpack(regs, spec.address)
 
-    def _write_register(self, address: int, value: int) -> None:
-        """Write a single Modbus register.
+    def _write_registers(self, address: int, value: int) -> None:
+        """Write to Modbus registers.
```

Comment:
These changes seem unrelated to "add more functions". They belong in their own commit.

Reply:
Best to ask for review on things that are finished, because I will need to review again after your fixups.